Library/PhotosController.swift

```diff
@@ -48,9 +47,31 @@ class PhotosController: UICollectionViewController {
             self.sections = Photo.constructRemoteElements()
             self.collectionView?.reloadData()
         }
+
+        #if os(tvOS)
+        let playPauseTapRecognizer = UITapGestureRecognizer(target: self, action: #selector(self.playPause(gesture:)))
+        playPauseTapRecognizer.allowedPressTypes = [NSNumber(value: UIPressType.playPause.rawValue)]
```

Comment:
Oke so .playPause is a astandard enum thngy that you turn unto an NSNumber and add to an array to set as `allowedPressTypes` 👍